### DISCUSSION TITLE
не работает chia wallet

### DISCUSSION THREAD

**Discussion by AndriyKovalenko83** [Chia-Network/chia-blockchain#18364] [Support]
Добрый день. Установил кошелек Chia lite. Отсутствует соединение кошелка с сетью. ион(кошелек) не синхронизируется.  У меня windows 10. Что делать? Спасибо.

**Comment by BrandtH22** [comment]
Hey @AndriyKovalenko83 , 

As a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

As for the issue can you please provide your log file from `~/.chia/mainnet/log/debug.log` ?

This has been translated using translation software and may contain errors.
---
В качестве предупреждения мы, как правило, можем предоставить более своевременную и тщательную поддержку на нашем сервере Discord (https://discord.gg/chia).

Что касается проблемы, не могли бы вы предоставить свой лог-файл из '~/.chia/mainnet/log/debug.log'? 

Оно было переведено с помощью программного обеспечения для перевода и может содержать ошибки.

**Reply by AndriyKovalenko83** [reply]
каким образом предоставить файл? Я не очень разбираюсь в програмах... Вам надо выслать сам текстовый документ или скопировать и вставить в сообщении его содержание? В этой папке один файл ._debug.lock - пустой, а другой debug - текстовый

**Reply by BrandtH22** [reply]
Yes please provide the file that has the text in it. It should be named debug.log. You can drag and drop the file into this chat to provide it. 

As a reminder we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia)
This has been translated using translation software and may contain errors.

Оно было переведено с помощью программного обеспечения для перевода и может содержать ошибки.
Да, пожалуйста, предоставьте файл, в котором есть текст. Он должен называться debug.log. Вы можете перетащить файл в этот чат, чтобы предоставить его. 

Напоминаем, что в целом мы можем обеспечить более своевременную и тщательную поддержку на нашем сервере Discord (https://discord.gg/chia)

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
